idea_town/frontend/static-src/app/templates/account-inactive-page.js

```diff
@@ -1,11 +1,16 @@
 export default `
-  <div class="page">
-    <section class="page main-content" data-hook="account-inactive-page">
-      <header data-hook="main-header"></header>
-      <h1>Account Inactive</h1>
-      <h2>We're not quite ready for you yet!<br>
-          Please come back in a few weeks.</h2>
+  <div class="page" id="full-page-notice">
+    <section class="main-content" data-hook="account-inactive-page">
+      <div class="vertical-centering-container">
+        <div id="beta-notice-modal" class="delayed-fade-in">
+          <h1>Thanks for signing up!</h1>
+          <h2>We'll let you know as soon as Idea Town is ready.</h2>
+          <div class="modal-controls">
+            <a href="/"><button class="button primary">Home</button></a>
```

Comment:
I spotted this in #267, but https://html5.validator.nu/ doesn't seem to like nesting `<button>` tags in an `<a>` element for unexplained reasons.

> **Error:** The element `button` must not appear as a descendant of the `a` element.
> From line 15, column 25; to line 15, column 55
> ```html
href="/"><button class="button primary">Home</
```

I tried shuffling some elements and classes in the dev tools locally, but all experiments looked pretty tragic, so we may need to adjust some CSS styling as well (in a future PR).